uk.co.compucorp.civicrm.hrcontactactionsmenu/phpunit.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
```

Comment:
Since this extension has tests now, please add it to the last of extension in the [Jenkinsfile](https://github.com/civicrm/civihr/blob/staging/Jenkinsfile#L293-L386). Before you do that, please sync your feature branch with staging, so it receives the latest updates that allow PRs to feature branches to be tested. Finally add the "ready to test" label to this PR